deploy/README.md

```diff
@@ -0,0 +1,158 @@
+# Kubernetes deployment
+
+A helm chart for installing OpenVino Model Server on Kubernetes cluster is provided. By default the cluster contains 
+a single instance of the server but the _replicas_ configuration parameter can be set to create a cluster 
+of any size, as described below. This guide assumes you already have a functional Kubernetes cluster and helm 
+installed (see below for instructions on installing helm).
+
+The steps below describe how to setup a model repository, use helm to launch the inference server and then send 
+inference requests to the running server. 
+
+## Installing Helm
+
+Please refer to: https://helm.sh/docs/intro/install for Helm installation.
+
+## Model repository
+
+If you already have a model repository you may use that with this helm chart. If you don't, you can use any model
+from https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin .
+
+OpenVINO Model Server needs models that it will make available for inferencing. For example you can 
+use Google Cloud Storage bucket:
```

Comment:
Based on feedback last week, please add AWS S3 and Azure Blob examples. https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction. (We have this already for S3, but should explain how to use: https://github.com/openvinotoolkit/model_server/blob/master/example_k8s/openvino_model_server_s3_rc.yaml)

Reply:
I can't add Azure Blob examples, because OVMS doesn't support this storage so far. I created a task for adding support for Azure storage: https://jira.devtools.intel.com/browse/CVS-31091 .

Going to add S3 instructions.

Reply:
Sounds good.